test_themes/tests/test_new_page_templates.py

```diff
@@ -172,7 +172,8 @@ def check(theme_name, website):
                             # Only look at blocks in website.snippets
                             html_tree = blocks_el[0]
                         for el in html_tree.xpath('//*[@class]'):
-                            classes = el.attrib['class'].split(' ')
+                            # Retrieves classes, excluding empty strings
+                            classes = [cls for cls in el.attrib["class"].split(" ") if cls]
```

Comment:
```suggestion
                            classes = el.attrib['class'].split()
```
Actually, can't it just be about removing the `' '` `split` param?

Reply:
Indeed, I push-forced to do that

Reply:
> Actually, can't it just be about removing the `' '` `split` param?

thanks, I didn't know split() was so different in Python compared to js